game/src/sandbox/dashboards/commuter.rs

```diff
@@ -274,12 +274,16 @@ impl CommuterPatterns {
                             .map(|(_, count)| count)
                             .unwrap_or(0);
                         let label_text = format!("{}", abstutil::prettyprint_usize(count));
-                        batch.append(
-                            Text::from(Line(label_text).fg(Color::BLACK))
-                                .render_to_batch(ctx.prerender)
-                                .scale(2.0)
-                                .centered_on(compare_to_block.shape.polylabel()),
-                        );
+                        let label = Text::from(Line(label_text).fg(Color::BLACK))
+                            .render_to_batch(ctx.prerender)
+                            .scale(2.0)
+                            .centered_on(compare_to_block.shape.polylabel());
+
+                        let dims = label.get_dims();
+                        let label_bg =  Polygon::rounded_rectangle(dims.width + 70.0, dims.height + 20.0, None);
+                        let bg = GeomBatch::from(vec![(Color::WHITE, label_bg)]).centered_on(compare_to_block.shape.polylabel());
```

Comment:
I'm kind of ambivalent about the pill shaped background - but we've got to do *something* to make the text legible. WDYT?

Reply:
I personally like it, even on nice large blocks. Even easier to see.